### PR TITLE
feat(*): suport testdata mock

### PR DESCRIPTION
**Pull request by jiuhuche120**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module github.com/TaiChiChain/data-producer
```
Comment (zunkk) [inline go.mod]: 更新下机构名

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
-# data-producer
\ No newline at end of file
+# data-producer
+
+A stress test tool for [@axiom](https://github.com/TaiChiChain/axiom)'s Ethereum JSON RPC🚀
```
Comment (zunkk) [inline README.md]: 改下机构名

File: go.mod (hunk 1 above)
Comment (jiuhuche120) [reply go.mod]: fixed

File: internal/utils/rpc.go [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+const AdminPrivateKey = "b6477143e17f889263044f6cf463dc37177ac4526c4c39a7a344198457024a2f" // axiom admin
+
+//const AdminPrivateKey = "c3363076b5b8f1418da917f0ff06eedaee654f347836bcebec1cd79614932984" // private eth admin
+
+const url = "http://localhost:8881" // axiom json rpc
```
Comment (zunkk) [inline internal/utils/rpc.go]: 这个有变更的需求么

File: README.md (hunk 2 above)
Comment (jiuhuche120) [reply README.md]: updated

File: go.mod (hunk 1 above)
Comment (jiuhuche120) [reply go.mod]: updated

File: internal/utils/rpc.go (hunk 3 above)
Comment (jiuhuche120) [reply internal/utils/rpc.go]: maybe load from command line is a better choice. The default url is http://localhost:8881, but also can changed by command args

**Pull request merged**